tests/languages/elisp/string_feature.test

```diff
@@ -0,0 +1,14 @@
+""
+"foo\
+bar"
+
+----------------------------------------------------
+
+[
+	["string", ["\"\""]],
+	["string", "\"foo\\\r\nbar\""]

```

Comment:
This leads to a failure which I don't understand
```
Token Stream:
[["string",["\"\""]],"\r\n\"foo\\\r\nbar\""]
-----------------------------------------
Expected Token Stream:
[["string",["\"\""]],["string","\"foo\\\r\nbar\""]]
-----------------------------------------
```

Reply:
My impression is that the second case is not recognized as a string. I'm not sure if it's a problem I'm how I'm writing the test cases or in the implementation

Reply:
I'm inclined to assume the implementation is right and my testing is wrong. If so, maybe the problem is that listing strings like this isn't valid lisp. Maybe the test tokens could me changed to 
```lisp
(
""
"foo
bar"
)
```

Reply:
Currently, your string pattern in the component allows only unescaped line feeds. https://github.com/PrismJS/prism/pull/1297#discussion_r168692063